### DISCUSSION TITLE
Printing from SEB for macOS

### DISCUSSION THREAD

**Discussion by peha2407** [SafeExamBrowser/seb-mac#352] [Ideas]
Hi folks
I have heard that it is possible to print exams under Windows - unlike under macOS. This is annoying for me as a member of a school where exams are taken on Macs, as the teachers also urgently want such an option. 
Will it be possible to print from macOS devices in a future version of SEB?
Regards,
Peter

**Comment by danschlet** [comment]
We always found it weird anybody would want to rely on such mechanical devices as printers to work correctly during a high-stress situation as exams. Also we are working for and favor **digital exams**, which don't waste resources as paper.

But ok, for feature parity and those who are not able to make the transition to fully digital we will look into this, when we have time. Unfortunately our development roadmap is very full already.

**Reply by peha2407** [reply]
Thanks for your reply! It is very important to our teachers that their students can also hand in their texts in printed form and that they can correct the exams on paper. I'm just the IT supporter ...

**Comment by danschlet** [comment]
Continue discussion in issue https://github.com/SafeExamBrowser/seb-mac/issues/353.

**Please don't open both a discussion and an issue**, an issue is enough for feature requests.
